packages/fx-core/src/component/provisionUtils.ts

```diff
@@ -208,7 +208,47 @@ export class ProvisionUtils {
     }
     return ok(undefined);
   }
-
+  // /**
```

Comment:
why not remove the code

Reply:
Will be used in the future when integrating provision